ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-netobserv-perf-tests__netobserv-aws-4.19-nightly-x86.yaml

```diff
@@ -39,6 +39,8 @@ tests:
     dependencies:
       EBPFAGENT_PR_IMAGE: ""
       FLP_PR_IMAGE: ""
+    dependency_overrides:
```

Comment:
@memodi is there a way to override the run name as it appears in the spreadsheet / mail ?
e.g. with some injected name within "node-density-heavy-25nodes-rehearse-65121-2025-05-20T10:19:20Z"
So we can triage mails more easily

Reply:
@jotak that's certainly possible, I filed a ticket https://issues.redhat.com/browse/NETOBSERV-2263